components/petdex/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,63 @@
+'use client'
+
+import { useState } from 'react'
+import { Logo } from './assets/Logo'
+import { MenuButton } from './assets/Menu-button'
+import { SearchIcon } from './assets/SearchIcon'
+import { Menu } from './assets/Menu'
+
+function Navbar() {
+  const [isOpen, setIsOpen] = useState(false)
+  const [isSearch, setIsSearch] = useState(true)
+
+  const handleButton = () => setIsOpen(o => !o)
+
+  const handleSearchButton = () => setIsSearch(s => !s)
+
+  return (
+    <div className="py-8">
+      <header className="flex  justify-between w-[80%] h-20 mx-auto sm:w-[90%]">
+        <div className="flex">
+          <nav className='flex items-center mobile:w-[90%] mx-auto '>
+            <div className='w-28 items-center mr-10'>
+              <Logo isSearch={isSearch} />
+            </div>
+            <div
+              className={`${!isOpen ? '' : 'left-[-400px]'} absolute flex flex-col left-0 top-[8rem] lg:relative lg:left-0 lg:top-0 `}
+            >
```

Comment:
acho que essa abordagem não é a das melhores usando tailwind, bom utilizar [clsx/tailwindmerge](https://korayguler.medium.com/how-to-merge-react-and-tailwind-css-class-names-f5faeb10ed24)

além do position mudar de acordo com o tamanho da tela, não me parece certo

Reply:
Vou le esse artigo, valeu